lib/maxwell/adapter/util.ex

```diff
@@ -136,23 +137,10 @@ defmodule Maxwell.Adapter.Util do
   end
   defp multipart_body(:end), do: :eof
 
-  defp append_query_string(url, path, query)when query == %{}, do: url_joint_path(url, path)
+  defp append_query_string(url, path, query)when query == %{}, do: Path.join(url, path)
```

Comment:
You should not use Path.join for it behaves differently for Linux and Windows. You will get your URL concatenated by `\` instead of `/`, which does not commit to the URL specification.